sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/models/RegistryArtifactOrderBy.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.containers.containerregistry.implementation.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for RegistryArtifactOrderBy. */
+public final class RegistryArtifactOrderBy extends ExpandableStringEnum<RegistryArtifactOrderBy> {
```

Comment:
Should this be made public API to make it easier to set this when listing?

Reply:
Not sure what you mean here - it is also present in public API.

---
In reply to: [609168232](https://github.com/Azure/azure-sdk-for-java/pull/20368#discussion_r609168232) [](ancestors = 609168232)